### ISSUE TITLE
Thank you!

### ISSUE THREAD

**Issue by walthowd** [ptr727/DWSpectrum-LSIO#6]
Just wanted to say thank you for a great docker image of DW Spectrum. I've been running DW Spectrum on bare metal and was able to move over to your Docker image seamlessly by just moving `etc` and `var` from DW's opt directory into my docker `/config` mount -- So much easier now to let Docker manage the system and the updates! Thanks again, you are doing a great job.

**Comment by walthowd** [comment]
Thanks!

**Comment by ptr727** [comment]
Glad you found it useful. I just heard from Nx today that the 4.1 version will allow a setting to specify filsystem types, this will make native usage on Unraid possible.

**Comment by walthowd** [comment]
That will be great! It's a great product but things like the weird file system checks and antiquated licensing scheme hold it back. 

I've been bugging the DW Spectrum sales guys to try to show them there is a market for home / small business use. And I think with some work they could compete against Blue Iris in that market -- but they are so focused on their niche in business/security old school VMS that I don't think they can see it.

**Comment by ptr727** [comment]
I totally agree, over the years I've tried several free and commercial solutions for my home IP cameras. When I moved most of my home services from VM's to docker, DW / NX was just so much easier and so much lighter on resources. I do wish they would start adding image recognition and automation capabilities, similar to what e.g. Google cameras offer.

**Comment by walthowd** [comment]
Yes, adding in tensorflow/tflite would be a no brainer as well!

For me, I have a lot of wireless cameras and I don't want to send multiple streams over the air. So I send all the streams to DW Spectrum first and then it proxies the streams to Frigate ( https://github.com/blakeblackshear/frigate ) and Home Assistant. This works surprisingly well, and I wish I knew the secret sauce that DW Spectrum bakes in to get so much less resource utilization. I believe they are using FFMPEG under the hood.

Frigate does real time tflite (with a Google coral TPU) for object detection and pushes notifications over MQTT back to Home Assistant.

**Comment by ptr727** [comment]
That's really cool, I run HA in docker, but I'm not actively using it, every time I get something working the next version of HA seems to break it ;)

**Comment by eddyg** [comment]
Hi Pieter & Howard! I stumbled across this old issue (and apologize for adding a comment to it, but it seemed like an easy way to engage you guys) while googling for "DW Spectrum" and "Home Assistant". I am one of those (apparently rare) people who wanted something _like_ Blue Iris, but wanted a solution that would run under Linux (ideally in Docker). A couple years ago I tried a bunch of IPVMS options (commercial and Open Source), and ended up rolling my own  Dockerfile and buying DW Spectrum licenses. 😉

Long story short: I was very happily surprised to find a maintained source for DWS Docker images, and _very_ interested in how Walt is "proxying streams from DWS to Frigate and Home Assistant". Do you have any config examples showing how you do this?!

Pieter: thanks so much for the Docker images, and Walt, thanks in advance for any info on your proxy setup to HA and Frigate!
